_includes/v22.2/cdc/privilege-model.md

```diff
@@ -0,0 +1,25 @@
+{{site.data.alerts.callout_info}}
+Starting in v22.2, CockroachDB introduces a new changefeed privilege model that provides finer control over a user's privilege to run changefeeds. 
+
+There is continued support for the [legacy privilege model](#legacy-privilege-model) in v22.2, however it **will be removed** in a future release. We recommend implementing the new privilege model that follows in this section for all changefeeds.
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v22.2" %} You can [grant](grant.html#grant-privileges-on-specific-tables-in-a-database) a user the `CHANGEFEED` privilege to allow them to create changefeeds on a specific table:
+
+{% include_cached copy-clipboard.html %}
+~~~sql
+GRANT CHANGEFEED ON TABLE example_table TO user;
+~~~
+
+This privilege model provides a more granular way to grant users the ability to create a changefeed on a table. A user granted the `CHANGEFEED` privilege can create changefeeds on the target table even if the user does **not** have the [`CONTROLCHANGEFEED` role option](alter-role.html#role-options) or the `SELECT` privilege on the table. 
+
+Since you can grant the `CHANGEFEED` privilege to a user or role **without** them needing the `SELECT` privilege on a table, these users will be able to create changefeeds, but they will not be able to run a `SELECT` query on that data directly. {% if page.name == "create-changefeed.md" %} However, these users could still read this data indirectly if they have read access to the [sink](changefeed-sinks.html), or create a "sinkless" changefeed that emits messages to the SQL session.{% else %} However, these users will be able to read this data indirectly as `EXPERIMENTAL CHANGEFEED FOR` emits changefeed messages to the SQL session. {% endif %}
+
+You can add `CHANGEFEED` to the user or role's [default privileges](security-reference/authorization.html#default-privileges) with [`ALTER DEFAULT PRIVILEGES`](alter-default-privileges.html#grant-default-privileges-to-a-specific-role):
+
+{% include_cached copy-clipboard.html %}
+~~~sql
+ALTER DEFAULT PRIVILEGES GRANT CHANGEFEED ON TABLES TO user;
+~~~
+
+Users with the `CONTROLCHANGEFEED` role option must have `SELECT` on each table, even if they are also granted the `CHANGEFEED` privilege.
```

Comment:
nit: this sentence is possibly unneeded, given that we mentioned this already on line 14. It also seems a bit abrupt following a demonstration of new syntax to discuss a limitation of old.

Reply:
Yes and no. I think line 14 is the opposite really, and if someone approaches privileges from the role option pov they will kind of want to know that if they then try and grant the new priv that's not going allow the user to run a cf without having first granted `SELECT` too. Not my favorite solution, but I have added this into a note box in the same position — moving it to the legacy section was not preferred especially as we're referencing the new privilege.

But it's a good point, that did feel a pretty abrupt way to end that section. Hopefully, this note box will bridge the worlds between legacy and shiny new.